<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmListSubtitles</class>
 <widget class="QDialog" name="frmListSubtitles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>343</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QNapi - wybierz napisy z listy</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icon/qnapi-48-icon.png</normaloff>:/icon/qnapi-48-icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>176</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lbSelectSubtitles">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Z poniższej listy wybierz napisy, które chcesz dopasować do pliku&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;film.avi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pbDownload">
     <property name="text">
      <string>Pobierz napisy</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pbDontDownload">
     <property name="text">
      <string>Nie pobieraj napisów dla tego filmu</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="lbGreenHint">
     <property name="text">
      <string>Na zielono podświetlono napisy &lt;b&gt;prawdopodobnie&lt;/b&gt; pasujące do Twojego filmu.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="lbRedHint">
     <property name="text">
      <string>Na czerwono podświetlono napisy &lt;b&gt;prawdopodobnie&lt;/b&gt; nie pasujące do Twojego filmu.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListWidget" name="twSubtitles">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbDownload</sender>
   <signal>clicked()</signal>
   <receiver>frmListSubtitles</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDontDownload</sender>
   <signal>clicked()</signal>
   <receiver>frmListSubtitles</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twSubtitles</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>pbDownload</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
